src/pages/Account.vue

```diff
@@ -10,190 +10,205 @@
                         v-bind:state="message.state"
                     ></message>
                 </transition>
+                <div v-if="spinnerActive">
+                    <spinner
+                        v-bind:active="spinnerActive"
+                        text="Please wait, working on it..."
+                    />
+                </div>
+                <div v-if="!spinnerActive">
```

Comment:
Could also be "v-else", which when applied to the next element after a "v-if", will act as an else for. It.

Reply:
Done